events_boilerplate.go

```diff
@@ -69,6 +69,12 @@ func (e *EventNewAddonType) TypeDataPtr() interface{} {
 func (e *EventUpgradeAddonType) TypeDataPtr() interface{} {
 	return &e.TypeData
 }
+func (e *EventDatabaseAddFeatureType) TypeDataPtr() interface{} {
```

Comment:
Shouldn't you add to this file the login related events?

Reply:
You should "Resolve" the conversations. The person who opened it is responsible for considering it closed.

You didn't implement `TypeDataPtr` functions for the events in the `events_account.go` file.

Reply:
Done through code generation